server/repository/mongorepo.py

```diff
@@ -80,6 +80,15 @@ def delete(self, shift_id):
         self.collection.delete_one({"_id": ObjectId(shift_id)})
         return
 
+    def delete_volunteer_shift(self, shift_id, user_id):
```

Comment:
Instead of adding this code for time being, you can just make comments about what should be done here.

Reply:
This has to be here it's the function that uses Python to remove the shift, Nhi didn't have to do it since the function she needed already is made. I don't think we should make someone reimplement it later, as it belongs here and does not impact the other back-end functionality.